services/movie.py

```diff
@@ -22,6 +27,7 @@ def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
```

Comment:
Consider handling the case where the movie with the given `movie_id` does not exist. You can use `get_object_or_404` or handle the `Movie.DoesNotExist` exception to provide a more user-friendly error message.